packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts

```diff
@@ -45,27 +44,25 @@ export class AuditLogManager {
     });
 
     this.logger.info('Getting audit logs (organizationId=%s, filter=%o)', organizationId, filter);
-    const where: SqlValue[] = [];
-    where.push(sql`organization_id = ${organizationId}`);
-
-    const from = formatToClickhouseDateTime(startOfDay(filter.startDate));
-    const to = formatToClickhouseDateTime(endOfDay(filter.endDate));
-    where.push(sql`timestamp >= ${from} AND timestamp <= ${to}`);
-
-    const whereClause = where.length > 0 ? sql`WHERE ${sql.join(where, ' AND ')}` : sql``;
```

Comment:
These can easily be inlined to simplify complexity and remove unnecessary condition checks that are always truthy (`where.length > 0`).